packages/common/components/ProposalList.tsx

```diff
@@ -22,8 +22,9 @@ import {
 import { useDaoInfoContext } from './DaoPageWrapper'
 import { SuspenseLoader } from './SuspenseLoader'
 
-// Max = 30
-const PROP_LOAD_LIMIT = 30
+// Contracts enforce a max of 30, though this is on the edge for DAOs with
```

Comment:
Let's add a wee bit more context at the top of this comment here - "The number of proposals a DAO will load on the frontend." Rest of the comment is great. 

Would be nice to double click on what it means for 30 to be "on the edge." Do you mean, DAOs with big proposals will  out-of-gas if their proposals are exceptionally big? How big is too big? Can we _ever_ pick some number where we'll never see an out of gas error?